src/components/nodes/boundaryMessageEvent/boundaryMessageEvent.vue

```diff
@@ -0,0 +1,29 @@
+<script>
+import BoundaryEvent from '@/components/nodes/boundaryEvent/boundaryEvent';
+import intermediateMailSymbol from '@/assets/intermediate-mail.svg';
+
+export default {
+  extends: BoundaryEvent,
+  mounted() {
+    this.shape.attr('image/xlink:href', intermediateMailSymbol);
+    this.shape.attr({
+      body: {
+        stroke: '#000',
+        strokeWidth: 1,
+        fill: '#fff',
+      },
+      body2: {
+        stroke: '#000',
+        strokeWidth: 1,
+        fill: '#fff',
+      },
+      image: {
+        width: 20,
+        height: 20,
+        y: 3,
+        x: 3,
+      },
+    });
+  },
```

Comment:
The attributes set in `this.shape.attr` appear to be the same ones used in `intermediateMessageCatchEvent.vue`; do you think you can extract them out into a separate file and use it in both places, to help eliminate duplication?